### ISSUE TITLE
Not working with Drupal 9.2

### ISSUE THREAD

**Issue by ravindrasingh22** [ibm-apiconnect/devportal#25]
Not supported with Vanilla Drupal. 

I have installed Drupal Vanila and tried to enable IBM APIC. (already installed Drupal) but it returns 

`Unable to install IBM APIC, comment.type.comment, core.entity_form_display.comment.comment.default, core.entity_view_display.comment.comment.default, core.entity_view_display.user.user.compact, core.entity_view_display.user.user.default, field.field.comment.comment.comment_body, field.field.user.user.user_picture, field.storage.node.comment, field.storage.user.user_picture, taxonomy.vocabulary.tags already exist in active configuration.
`

Also. all module dependencies have been installed via composer. I am using Drupal Docker image from d.o.  

Basically, all the above configuration comes with Drupal core default so it should not be a part of Module to recreate those.

**Comment by chrisdudley** [comment]
We do not support installing our modules on vanilla drupal.
You can only use them when running as part of the APIC Developer Portal.

What you are attempting to do will never work - the portal requires an API Manager in order to authenticate with, so you won't even be able to login.

**Comment by ravindrasingh22** [comment]
ohk, got it. but there are some modules which are not more up to date. e.g Votingapi Widgets
 and it throw errors like   - drupal/core 8.0.0-beta6 requires doctrine/common dev-master#a45d110f71c323e29f41eb0696fa230e3fa1b1b5 -> found doctrine/common[2.1.3, ..., 2.13.x-dev, 3.0.0, ..., 3.2.x-dev] but it does not match the constraint.

**Comment by chrisdudley** [comment]
Use the composer.lock and composer.json files from the apim_profile directory and you'll get the exact module versions that are included in the APIC Developer portal.
Just copy them into your local deployment and then run composer update.

**Comment by ravindrasingh22** [comment]
Checked composer.json file, it asks for username and password for {
      "type": "composer",
      "url": "https://eu.artifactory.swg-devops.com/artifactory/api/composer/apic-dev-portal-composer-local"
    },

Was this mandatory to use in this way?

**Comment by chrisdudley** [comment]
No, you wont have access to that, just remove that section from the file. Its not supposed to be there :-)

**Comment by ravindrasingh22** [comment]
the composer.json does not sounds stable, after updating it it, during  installation it still asks for missing libraries
MASONRY LIBRARY MISSING
File Browser requires the Masonry library. Download the newest release from https://github.com/desandro/masonry/releases and place it in /libraries
DROPZONE LIBRARY MISSING
Dropzonejs requires the dropzone.min.js library. Download it (https://github.com/enyo/dropzone) and place it in the libraries folder (/libraries)
IMAGESLOADED LIBRARY MISSING
File Browser requires the imagesLoaded library. Download the newest release from https://github.com/desandro/imagesloaded releases and place it in /libraries

**Comment by chrisdudley** [comment]
I told you this isn't going to work. Not only will it not work, but even if it did work it wouldn't be of any real use in actually doing custom module / theme development for the APIC Developer Portal. (I assume that's your goal)
I'm not sure why you don't believe me.

If you want all the prereqs to be provided then use our Developer Portal images.
Please stop posting insinuations that it doesn't work with a specific drupal version or its not stable. Its perfectly stable, you're just trying to do something I have explicitly told you isn't going to work.

We do not support and are not going to assist in what you are attempting to do, I really don't think there is anything further to be gained from posting on this issue.
